Makefile

```diff
@@ -10,15 +10,18 @@ HELM_REPO ?= $(CURDIR)/repo
 all: clean lint build deploy
 
 # Ensure the Helm chart and its metadata are valid
-lint: helm $(SHIM_DIR)
+lint: helm
 	@echo "== Linting Chart..."
-	@helm lint $(SHIM_DIR)
+	@git remote add traefik https://github.com/containous/traefik-helm-chart >/dev/null 2>&1 || true
+	@git fetch traefik master >/dev/null 2>&1 || true
+	@docker run --rm -t -v $(CURDIR):/charts -w /charts quay.io/helmpack/chart-testing:v3.0.0-beta.1 \
```

Comment:
This puts a docker requirement for `make lint`... is that something that we feel is acceptable?

Kind of nice to be able to run a quick lint without docker running.

Reply:
Good point. 

=> The requirements carried by `ct`'s docker image are listed here: https://github.com/helm/chart-testing#prerequisites . It looks acceptable for me to require it, and fail fast (e.g. the Makefile fails with a simple error message if the requirement is missing).


=> We could also use a variable to use or not docker, likewise the traefik build makefile also.

Reply:
Oh, I realize that I've put the `docker` make target, which ensure that it is a requirement, is in #60 while it should be here. Let me transplant it here, so the change will be atomic.

Reply:
@dtomcej running `make lint` without Docker is now possible, with requirements check, and documented.